[2016-01-11 07:57:01,488] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:57:01,600] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:57:01,602] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:57:09,442] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:57:10,021] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:57:10,022] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:57:10,128] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:57:10,129] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:57:10,236] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:57:10,238] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:57:54,121] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:57:54,598] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:57:54,599] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:57:54,695] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:57:54,696] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:57:54,779] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:57:54,780] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:58:00,514] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:58:00,900] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:58:00,901] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:58:00,976] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:58:00,977] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:58:01,057] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:58:01,058] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:58:02,567] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:58:03,078] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:58:03,079] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:58:03,188] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:58:03,189] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:58:03,290] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:58:03,291] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:58:14,747] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:58:16,291] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:58:16,292] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:58:16,489] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:58:16,490] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:58:16,612] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:58:16,614] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:58:18,623] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:58:19,067] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:58:19,070] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:58:19,149] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:58:19,150] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:58:19,238] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:58:19,239] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:58:23,966] {main.py-><module>:135} INFO - 



******************[93m2016-01-11 07:58:23.966264[39m*******************
[2016-01-11 07:58:24,540] {main.py-><module>:135} INFO - 



******************[93m2016-01-11 07:58:24.540621[39m*******************
[2016-01-11 07:58:26,794] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:58:27,291] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:58:27,293] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:58:27,381] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:58:27,382] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:58:27,473] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:58:27,474] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:58:55,958] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:58:56,511] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:58:56,512] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:58:56,691] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:58:56,692] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:58:56,856] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
